pythoncalculator/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .add import add 
```

Comment:
Should this be * instead of +?

Reply:
Thanks for checking that.  Yes, it should be *.  M